cmd/goConvert/goConvert.go

```diff
@@ -204,7 +204,7 @@ func main() {
 	}
 
 	// get logger
-	err := logging.Init("goConvert", version.Short(), "debug", "console")
```

Comment:
It would make sense, but leaves the caller to take care of translating `string` (from config) to a `logging.Level`, which is what I wanted to spare them from.

As for `name`: should be an optional parameter IMO. The version as well.

Reply:
As discussed
- [ ] provide function `logging.LevelFromString(lvl string) logging.Level`
- [ ] provide encoding as `logging.Encoding` constants